xcp_d/interfaces/concatenation.py

```diff
@@ -279,12 +266,7 @@ class _ConcatenateInputsInputSpec(BaseInterfaceInputSpec):
         mandatory=True,
         desc="TSV files with high-motion outliers indexed.",
     )
-    uncensored_denoised_bold = traits.List(
-        File(exists=True),
-        mandatory=True,
-        desc="Denoised BOLD data.",
-    )
-    interpolated_filtered_bold = traits.List(
+    denoised_interpolated_bold = traits.List(
```

Comment:
nitpick: For these I've been using `InputMultiObject` instead of `traits.list`

Reply:
oh interesting! Now that I think of it, I don't think I use this anywhere where there might be only one input